flowzone.yml

```
@@ -488,13 +488,22 @@ jobs:
       - name: Check for package.json
         id: npm
         run: |
+          runner_os="$(echo "${RUNNER_OS}" | tr '[:upper:]' '[:lower:]')"
           if test -f "package.json"
           then
             echo "found package.json"
-            echo "enabled=true" >> $GITHUB_OUTPUT
-            echo "private=$(jq -r '.private' package.json)" >> $GITHUB_OUTPUT
-            echo "docs=$(jq -r '.scripts | has("doc")' package.json)" >> $GITHUB_OUTPUT
-            echo "NODE_VERSIONS=[]" >> $GITHUB_ENV
+            os="$(cat < package.json | yq e .os[])"
```

```suggestion
            os_count="$(jq '.os | length' package.json)"
```
Nitpick, but why use yq to check the value of a json structure?

---

Yeah, that's because I am an idiot and constantly mix YAML and JSON. Thank you.